das/deadline_wrapper.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package das
+
+import (
+	"context"
+	"time"
+
+	"github.com/offchainlabs/nitro/arbstate"
+)
+
+type DeadlineWrapper struct {
```

Comment:
The definitions around contexts typically use "timeout" instead of "deadline".   So probably better to call this `TimeoutWrapper`.

Reply:
I had used the term deadline because it uses `context.WithDeadline` so I thought this would be better understood than timeout, which we are already using to mean the data expiry timeout. I can change it if you still think we should.

Reply:
Hah, I just looked at the docs and there is also a `context.WithTimeout` which does exactly what I want (`time.Now().Add(...)`), so I'll change it to that and change the name.